frontend/src/Judge/components/Shows.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import ShowCard from '../components/ShowCard'
+
+const NoShowsContainer = styled.div`
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Haven't looked at this visually, but a pro-tip, with flexbox, centering things is _real_ easy:

```css
display: flex;
align-items: center;
justify-content: center;
```

[Reference](https://philipwalton.github.io/solved-by-flexbox/demos/vertical-centering/)

Reply:
That only horizontally aligns it: 
![image](https://user-images.githubusercontent.com/6333152/36755047-c53a642c-1bbf-11e8-8028-dda4a22e9e7b.png)
vs the vertical allignment you asked for: 
![image](https://user-images.githubusercontent.com/6333152/36755078-e1e39ac6-1bbf-11e8-84db-1fdb16bcf83a.png)